vnext/ReactUWP/Base/UwpReactInstance.cpp

```diff
@@ -215,12 +216,14 @@ std::vector<facebook::react::NativeModuleDescription> GetModules(
             std::move(appTheme));
       },
       messageQueue);
-
+  modules.emplace_back(
+      AlertModule::name,
+      []() { return std::make_unique<AlertModule>(); },
+      messageQueue);
   modules.emplace_back(
       ClipboardModule::name,
       []() { return std::make_unique<ClipboardModule>(); },
       messageQueue);
-
   modules.emplace_back(
```

Comment:
Maybe add the spacing here, look like that's the style for most of them. You can add it above too.